app/scripts/configs/tracks-info.js

```diff
@@ -1702,7 +1702,8 @@ export const TRACKS_INFO = [
     name: 'BED-like track',
     thumbnail: null,
     availableOptions: [
-      'fillColor',
```

Comment:
My mistake.

Snippet from Slack conversation for the record:

pete 2:50 PM
On the fillColor parameter, do you have thoughts on whether to remove it? There’s currently two modes of operation:
No strand information: it doesn’t make sense to say plusStrandColor because items aren’t necessarily on a strand, therefore fillColor should be used
+ / - strand information: fillColor as a default, each strand can set using [plus/minus]StrandColor individually.
Does it make sense to keep it and use it like that?

new messages
Fritz 2:55 PM
I am more confused as to wether it's still there or not
I'd definitely keep it
What you describe above makes complete sense
I was just worried it got removed
pete 2:58 PM
:+1:
yeah, it looks like I removed it from availableOptions which was a mistake but it’s definitely still used by the renderer